### DISCUSSION TITLE
Wake model for layout optimization

### DISCUSSION THREAD

**Discussion by Rodrig03** [NatLabRockies/floris#580] [Q&A]
Hello again,

I'm running a layout optimization for a windfarm consisting on 30+ turbines using `LayoutOptimizationPyOptSparse` and the `SLSQP` solver.
I was testing the optimization for 100 iterations using the **Gauss Curl Hybrid** model and comparing it when **turbopark** was used as the wake model. The same initial conditions are being used for both models.

The optimization using the **Gauss Curl Hybrid** model produced some good-looking results, but the one using **turbopark** was not so efficient and the resulting optimization is not so good-looking.

Here you can see a plot of the generated optimization history (.out) for both cases.
![image](https://user-images.githubusercontent.com/122991766/219302031-dc51f87e-acab-46ee-acbe-00cf363d5f86.png)

Is there any reason why the optimization doesn't seem to work so well when **turbopark** is used as the wake model?
Is there a favorite wake model to be used for layout optimization?

Thanks a lot in advance.

**Comment by cherifM** [comment]
Hi @Rodrig03 ,
im doing similar things, 
please provide your floris yaml config file,  I may help you to figure out where is the issue.  But I have 2 questions 
1- why gradient based SLSQP method? I think its better if you use genetic based like nsga2 or  ALPSO both better optimizers in my opinion (global optimizer wich dosent have any locality issue), SLSQP in pyoptspare is very old and can sometimes violate the boundaries, i did try it. You should use gradient based only if you have nice initial layout!

2- how you are plotting the history (.out) ?  can you provide your script for plotting only,
best regards , 

cherif
